src/components/DpButtonRow/DpButtonRow.vue

```diff
@@ -6,15 +6,15 @@
       v-if="primary"
       :busy="busy"
       :data-cy="`${dataCy}:saveButton`"
-      :disabled="disabled"
+      :disabled="disabledPrimary"
```

Comment:
This is a nice improvement! I'm wondering though if we want to keep the option to disable both buttons easily by setting `disabled` to `true`, and if we want to disable just one button we could pass `disabled: { primary: true }` or something like that? Maybe it adds unnecessary complexity. What do you think? Do you know what the most common use case is?
Also, if we keep `disabledPrimary` and `disabledSecondary`, I think we need to adjust usages in demosplan?

Reply:
I decided to go with your approach. This way we also don't have to worry about breaking things eg. in addons. I also added the option to also pass the secondary button. It probably is not necessary, but I think it does not hurt. 
Done in [6ddeba1](https://github.com/demos-europe/demosplan-ui/pull/1183/commits/6ddeba1f334f03589c23efa9e0d2de3a2507ef43)